lib/Elevate/Blockers/AutoSSL.pm

```diff
@@ -0,0 +1,43 @@
+package Elevate::Blockers::AutoSSL;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Blockers::AutoSSL
+
+Blocker to check if Sectigo is the AutoSSL provider.
+
+=cut
+
+use cPstrict;
+
+use Cpanel::SSL::Auto ();
+
+use parent qw{Elevate::Blockers::Base};
+
+sub check ($self) {
+
+    return $self->_check_autossl_provider();
+}
+
+sub _check_autossl_provider ($self) {
+
+    my @providers = Cpanel::SSL::Auto::get_all_provider_info();
+
+    foreach my $provider (@providers) {
+        next unless ( ref $provider eq 'HASH' && $provider->{enabled} );
+
+        if ( defined $provider->{display_name}
+            && $provider->{display_name} =~ /sectigo/i ) {
+            return $self->has_blocker( <<~"EOS");
+            Unsupported AutoSSL provider: $provider->{display_name}
+            Move all accounts to a different AutoSSL provider before running elevate.
```

Comment:
Is there an API call that we can provide the customer to make it easy for them to do this / would `set_autossl_provider` work for this purpose?

Reply:
```set_autossl_provider``` should do it.  The might also need to invoke ```start_autossl_check_for_all_users```; but, I think that happens automatically on a regular basis.